### PR TITLE
Initial test structure

### PR DESCRIPTION
**Pull request by bernardoVale**
Please confirm if this is the desired/acceptable approach to handle edge cases like 404/5xx and also review the initial test structure for commands.

### CODE REVIEW

File: synth_tools/commands/agents.py [hunk 1]
```diff
@@ -38,8 +39,16 @@ def get_agent(
     """
     api = ctx.find_object(APIs)
     for i in agent_ids:
+        try:
+            a = api.syn.agent(i)
+        except KentikAPIRequestError as e:
+            if e.response.status_code == 404:
+                typer.echo(f"Agent {i} does not exists")
```
Comment (mmac-m3a) [inline synth_tools/commands/agents.py:46]: In both failure cases we should call `fail`, so that `synth_ctl` returns non-zero exit code. I do realize that we may not print results for all of the agents that actually are available, but I think it is more important to give the right signal that something was missing, especially for a tool parsing the output.
Also typo in the message: "exist"

File: synth_tools/tests/commands/test_agents.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+
+def fake_req(fixture_file):
+    d = Path(__file__).parent / fixture_file
+    with open(d) as stream:
+        return json.load(stream)
+
+
+@pytest.mark.parametrize(
+    "cmd, status_code, expected_in_output, not_expected_in_output, requests",
```
Comment (mmac-m3a) [inline synth_tools/tests/commands/test_agents.py:28]: It is little bit misleading to call the last field "requests", because it actually contains mocked response payload. We may eventually want to also check that `synth_ctl` is sending correct request.

This may come handy when prepping `synth_tools` for a new version of the API before it is even deployed. The modified version of `synth_tools` may then be used for testing the new version of the API. As a matter of fact, it would come handy right now :-). Han has new version of the API (beta2) ready to test, but he does not have any comprehensive enough test framework. Obviously, `synth_tools` would need to be modified to support new API schema and unit tests would be the only way to test it right now.

File: synth_tools/commands/agents.py [hunk 3]
```diff
@@ -38,8 +39,16 @@ def get_agent(
...
+            else:
+                typer.echo(f"Got unexpected response from API:\n{e.response.text}")
```
Comment (mmac-m3a) [inline synth_tools/commands/agents.py:48]: Print the whole text version of the exception: `fail(f"{e}")`. It contains also the request and status code, which may come handy when reporting the error to Kentik support.
